src/Microsoft.AspNetCore.Dispatcher.Abstractions/Endpoint.cs

```diff
@@ -6,7 +6,7 @@
 
 namespace Microsoft.AspNetCore.Dispatcher
 {
-    [DebuggerDisplay("{DisplayName,nq}")]
+    [DebuggerDisplay("{GetType()}{DisplayName,nq}")]
```

Comment:
put a space here. this is going to look like `Some.Type.Namedkfdfdkbydk`

I'd suggest formatting like like `{GetType().Name} - '{DisplayName,nq}'` or something similar. Make it look reasonable